modules/social_features/social_event/asyncapi.yml

```diff
@@ -32,8 +32,17 @@ channels:
                       type: string
                       description: The label of the event.
                     visibility:
-                      type: string
-                      description: The visibility of the event content.
+                      type: object
+                      properties:
+                        type:
+                          type: string
+                          description: The visibility type.
+                        group:
```

Comment:
@nkoporec As I can see we should have `groups` instead of `group` here.
Can you fix it please?